### PR TITLE
Add brexit checker support form

### PR DESCRIPTION
**Pull request by emmabeynon**
For https://trello.com/c/bgtCMkNP/243-zendesk-content-change-request-form

This creates a request form for departments to request a change to an existing action or creating a new action for the Get Ready For Brexit Checker.  Once submitted, a request is made to the Zendesk API to create a ticket.

brexit_check_form

### CODE REVIEW

File: spec/features/get_ready_for_brexit_checker_request_spec.rb [hunk 1]
```diff
@@ -11,7 +11,21 @@
...
+      "requester" => hash_including("name" => "John Smith", "email" => "john.smith@agency.gov.uk"),
+      "comment" => {
+        "body" =>
+"[Change title]
+Action title
+
+[Change description]
+Action description"
+      }
+    )
```
Comment (theseanything) [inline spec/features/get_ready_for_brexit_checker_request_spec.rb]: I think this needs indentation.

File: app/views/get_ready_for_brexit_checker_requests/_request_details.html.erb [hunk 2]
```diff
@@ -29,4 +29,57 @@
...
+    :add_lead_time,
+    label: "Lead time",
+    hint: "How long does the process take, if applicable?",
+    ) %>
+
+  <%= f.input(
+    :add_deadline,
+    label: "Deadline",
+    hint: "Does the user need to act before a certain date?",
+    ) %>
```
Comment (theseanything) [inline app/views/get_ready_for_brexit_checker_requests/_request_details.html.erb]: Do we need an input for priority?

File: spec/features/get_ready_for_brexit_checker_request_spec.rb (hunk 1 above)
Comment (emmabeynon) [reply spec/features/get_ready_for_brexit_checker_request_spec.rb]: I had originally thought so too, but it seems if you indent then it affects the API request that's made, so you get extra whitespace, like this:
`"comment"=>{"body"=>"[Action to change]\n          Action title\n\n          [Description of change]\n          Action description"`

When it should be this:
`"comment":{"body":"[Action to change]\nAction title\n\n[Description of change]\nAction description"`

There is precedent for this in the app, for example: https://github.com/alphagov/support/blob/master/spec/features/eu_exit_business_readiness_request_spec.rb#L21

File: app/views/get_ready_for_brexit_checker_requests/_request_details.html.erb (hunk 2 above)
Comment (emmabeynon) [reply app/views/get_ready_for_brexit_checker_requests/_request_details.html.erb]: No, the default in Zendesk is 'normal' and we don't have a requirement to change it to another level.

Comment (theseanything) [reply app/views/get_ready_for_brexit_checker_requests/_request_details.html.erb]: Ah sorry, I mean for the new action.

Comment (emmabeynon) [reply app/views/get_ready_for_brexit_checker_requests/_request_details.html.erb]: Ah I see - will check with Ben.

Comment (emmabeynon) [reply app/views/get_ready_for_brexit_checker_requests/_request_details.html.erb]: Priority field has been added 👍

**Pull request merged**